### DISCUSSION TITLE
How long will it take for the release of World 3 to Story Mode?

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2760] [Q&A] [answered]
I wondered since I don't know how long it's going to take for the third world to go into story mode. The World 2 demo arrived in update 0.6.0. And the final version with Corrupted Forest, which was previously called Ghostforest, came in 0.6.1. Then.  Will. it come in a possible 0.7.1 or 0.8.0?

**Comment by Frostwithasideofsalt** [comment, marked as answer]
World 3 comes in 0.8
